telegrambots-abilities/src/main/java/org/telegram/abilitybots/api/bot/DefaultAbilities.java

```diff
@@ -236,6 +237,25 @@ public Ability backupDB() {
         .build();
   }
 
+    /**
+     * The ability to cancel operation.
+     * User can use this method to cancel the current session and delete related chat.
+     * Usage: <code>/cancel</code>
+     *
+     * @return the ability to cancel operation
+     */
+    public Ability cancel() {
+        return builder()
+                .name(CANCEL)
+                .info("Cancel current operation.")
```

Comment:
Don't supply info as that would make it be reported in the commands report ability. Do add it to the wiki where appropriate.

Without info set, this ability will still be reported using the /commands operation. But, /report won't.